### ISSUE TITLE
Missing decoder(v4l2h264dec) problem for VPU example

### ISSUE THREAD

**Issue by mcanKayra** [toradex/torizon-samples#33]
There seems to be a problem with VPU example on bookworm-new branch. I built an image from the example Dockerfile for apalis-imx8qm and some plugins especially `v4l2h264dec` and `imxvideoconvert_g2d` are missing. Test script `vputests.sh` uses these plugins. I have already tried belows;

- Try with multiple images by changing tags from torizon/wayland-base-vivante (3, 3.2, 3.3 etc.)
- Set MACHINE arg with `ARG MACHINE=apalis-imx8` 
- Try with `--enable-vpu` tag while configuring `# imx-codec` section.

 I have followed this article
[https://developer.toradex.com/torizon/application-development/use-cases/multimedia/how-to-use-gstreamer-on-torizoncore/#imx8-vpu-accel](url)

**Comment by leograba** [comment]
@mcanKayra, thanks for your report.

Please forward this report to https://community.toradex.com, where our support team will follow it up.

Meanwhile, feel free to open a PR if you find a solution and would like to contribute to it.

**Comment by mcanKayra** [comment]
Based on your advice, I opened a topic in the forum, and one of the support team members mentioned that they followed the link and everything was working properly. When I checked again, I noticed that the volume bindings and device group rules in docker-compose.yml file were different from the docker run command I had been using for testing. My problem was solved when I used the settings in the compose file. So sorry, it is my bad, we can close the issue.

By the way it might make a small contribution, friend on the support team said:
`Also in the Dockerfile set BASE_VERSION to 3.3 This version works the best for accelerated VPU at the moment.`

Best regards.
